test-biblatex.tex

```diff
@@ -16,6 +16,7 @@
 
 \RequirePackage[utf8]{inputenc}
 \RequirePackage[T1,T2A]{fontenc}
+\RequirePackage{textcomp}
```

Comment:
Why this requirement was added?
Are there any symbols that aren't available without it?